etcdmain/etcd.go

```diff
@@ -62,6 +62,10 @@ func Main() {
 
 	var stopped <-chan struct{}
 
+	if cfg.name != defaultName {
```

Comment:
Or the name could be really set to default and the user has a valid initial cluster list.